docs/stable/1-client.md

```diff
@@ -259,9 +269,9 @@ For transaction support, Status clients currently have to rely on Infura.
 
 Status clients currently do not offer native support for file storage.
 
-#### Why do you use Whisper?
+#### Why do you use Waku?
 
-Whisper is one of the [three parts](http://gavwood.com/dappsweb3.html) of the
+Waku is one of the [three parts](http://gavwood.com/dappsweb3.html) of the
```

Comment:
I guess this should remain Whisper.

Reply:
Yeah. I wasn't sure if we were treating Waku as the direct successor and replacement for Whisper, but it doesn't make sense in this sentence. I think it would be best to revert the change and remove this sentence when we phase Whisper our 100%.